keep-ui/app/(keep)/alerts/alert-table-server-side.tsx

```diff
@@ -0,0 +1,494 @@
+import { useCallback, useEffect, useMemo, useRef, useState } from "react";
```

Comment:
is there `alert-table-client-side`? if not - why not using this as `alert-table`?